governance/default.go

```diff
@@ -521,18 +521,35 @@ func (g *Governance) ClearVotes(num uint64) {
 // parseVoteValue parse vote.Value from []uint8 to appropriate type
 func (g *Governance) ParseVoteValue(gVote *GovernanceVote) (*GovernanceVote, error) {
 	var val interface{}
-	k := GovernanceKeyMap[gVote.Key]
+	k, ok := GovernanceKeyMap[gVote.Key]
+	if ok == false {
```

Comment:
```suggestion
	if !ok {
```
Very minor comment

Reply:
sorry for the late. Thanks, reflected at https://github.com/klaytn/klaytn/pull/1110/commits/680be2ebb4e2fc84f7ecab4319657906761f9377.